### DISCUSSION TITLE
Support `--watch` flag for `vite:build`

### DISCUSSION THREAD

**Discussion by jwnx** [remix-run/remix#8767] [Proposals]
It's a supported `vite` flag and would allow us to use vite the same way we use remix server, i.e: `remix vite:build --watch` + `remix-serve ./build/server/index.js`.

**Comment by csoylu** [comment]
I am using cloudflare d1 and in order to use the d1 database in local, and to have a similar architecture, i have to launch my remix app from wrangler.
Wrangler does not compile my application but only serves it and updates if the build files change.
I need in this case to be able to do a `remix vite:build --watch` followed by a `wrangler pages dev`

**Comment by brophdawg11** [comment]
This discussion is being closed as part of the Remix v3 repository cleanup.

Remix v1/v2 was [upstreamed into React Router v7](https://remix.run/blog/wake-up-remix), so if this proposal is still relevant, please reopen or continue the discussion in the [React Router](https://github.com/remix-run/react-router) repository.
